src/appleseed/renderer/modeling/project-builtin/cornellboxproject.cpp

```diff
@@ -1014,6 +1017,43 @@ auto_release_ptr<Project> CornellBoxProjectFactory::create()
         // Attach the camera to the scene.
         scene->cameras().insert(camera);
     }
+    
+    //
+    // Environment
+    //
+
+    // Color called "black_radiance" is created and inserted into the scene.
+    static const float black_radiance[] = { 0.0f, 0.0f, 0.0f };
+    scene->colors().insert(
+        ColorEntityFactory::create(
+            "black_radiance",
+            ParamArray()
+                .insert("color_space", "srgb")
+                .insert("multiplier", "0.5"),
+            ColorValueArray(3, black_radiance)));
+
+    // Environment EDF called "black_edf" is created and inserted it into the scene.
+    scene->environment_edfs().insert(
+        ConstantEnvironmentEDFFactory().create(
+            "black_edf",
+            ParamArray()
+                .insert("radiance", "black_radiance")));
+
+
+    // Environment Shader called "black_shader" is created and insert it into the scene.
+    scene->environment_shaders().insert(
+        EDFEnvironmentShaderFactory().create(
+            "black_shader",
+            ParamArray()
+                .insert("environment_edf", "black_edf")));
+
+    // Environment called "black" is created and inserted to the scene.
+    scene->set_environment(
+        EnvironmentFactory::create(
+            "black",
+            ParamArray()
+                .insert("environment_edf", "black_edf")
```

Comment:
An environment EDF is the part of the environment that emits light. We don't want to emit any light (and in fact you may see a warning in the console since the environment is black). Don't set `environment_edf` and you'll be good to go.

Reply:
done